Bugfix for version 0.0.2 (reported by ecki@vdrportal):
The setup options for replacing VDR's original Timers and Schedule menu
are not remembered after a reboot.

--- remoteosd.c.orig	2007-03-19 12:58:42.000000000 +0100
+++ remoteosd.c	2007-03-19 12:59:27.000000000 +0100
@@ -108,6 +108,10 @@
 {
   if (!strcasecmp(Name, "HideMainMenuEntry"))
 	  RemoteOsdSetup.hideMainMenuEntry = atoi(Value);
+  else if (!strcasecmp(Name, "ReplaceSchedule"))
+	  RemoteOsdSetup.replaceSchedule = atoi(Value);
+  else if (!strcasecmp(Name, "ReplaceTimers"))
+	  RemoteOsdSetup.replaceTimers = atoi(Value);
   else if (!strcasecmp(Name, "ServerIp"))
 	  strn0cpy(RemoteOsdSetup.serverIp, Value, sizeof(RemoteOsdSetup.serverIp));
   else if (!strcasecmp(Name, "ServerPort"))
